docs/integrations/sources/hubspot.md

```diff
@@ -64,18 +64,18 @@ Depending on the type of engagement, different properties will be set for that o
 #### Requirements
```

Comment:
Can you split the getting started section and have one specifically for Airbyte cloud, and one specifically for OSS? The OSS section can be exactly the same as this, but Cloud should always recommend using oauth